modules/otel-collector-components.adoc

```diff
@@ -194,22 +194,112 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
           ...
 ----
 <1> The OpenCensus endpoint. If omitted, the default is `+0.0.0.0:55678+`.
-<2> The server-side TLS configuration. See the OTLP receiver configuration section for more details.
+<2> The server-side TLS configuration. See the OTLP Receiver configuration section for more details.
 <3> You can also use the HTTP JSON endpoint to optionally configure CORS, which is enabled by specifying a list of allowed CORS origins in this field.
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="kubernetesevents-receiver_{context}"]
+=== Kubernetes Events Receiver
+
+The Kubernetes Events Receiver collects events from the Kubernetes API server. The collected events are converted into logs.
+
+
+.OpenShift Container Platform permissions required for the Kubernetes Events Receiver
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+  labels:
+    app: otel-collector
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  - namespaces/status
+  - nodes
+  - nodes/spec
+  - pods
+  - pods/status
+  - replicationcontrollers
+  - replicationcontrollers/status
+  - resourcequotas
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
+  - deployments
+  - replicasets
+  - statefulsets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - extensions
+  resources:
+  - daemonsets
+  - deployments
+  - replicasets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  - cronjobs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+    - autoscaling
+  resources:
+    - horizontalpodautoscalers
+  verbs:
+    - get
+    - list
+    - watch
+----
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes Event Receiver
+[source,yaml]
+----
+  serviceAccount: otel-collector # <1>
+  config: |
+    receivers:
+      k8s_events:
+        namespaces: [project1, project2] # <2>
+    service:
+      pipelines:
+        logs:
+          receivers: [k8s_events]
+----
+<1> The service account of the Collector that has the required ClusterRole `otel-collector` RBAC.
```

Comment:
```suggestion
<1> The service account of the Collector that has the required cluster role `otel-collector` RBAC.
```

Reply:
Good to have ..

Reply:
I looked it up at the time. "ClusterRole" is used in the OpenShift docs as-is. For example: https://docs.openshift.com/container-platform/4.14/rest_api/rbac_apis/clusterrole-rbac-authorization-k8s-io-v1.html